### ISSUE TITLE
 Nvda screenreader is not announcing any status message after performing a search operation on environment and dynamics 365 combo box search control.

### ISSUE THREAD

**Issue by venugopalreddyp** [select2/select2#6161]
When user search the Environment (org) and Dynamics 365 app in combo boxes  ,search results not announcing the results count.
Repro steps:
1.Hit the URL "https://teams.microsoft.com/"
2.Tab till "Use the web app instead" link and press enter 
3.Tab till "+"(Add a tab) button and press enter.
4.Tab till "Dynamics 365 (int)" and press enter
5.Tab till the edit  the environment and dynamics 365 app
6.search the environment and dynamics 365 app names.
7.Observe that the screen reader is not announcing any status message after search operation.

Expected:  Screen reader must be announce the search results count.

**Comment by kevin-brown** [comment]
Unfortunately I'm not able to reproduce the issue within Microsoft Teams because I'm in the portion of world where Microsoft Teams is not used. With that being said, I'm fairly confident this is a duplicate of https://github.com/select2/select2/issues/3735.

Please check that ticket and I can merge these together if it is in fact a duplicate.

**Comment by venugopalreddyp** [comment]
This issue and issue id: #3735  both are not same. 

Repro steps:
1.Login the Microsoft teams
2.Select any teams and select general and click on + button
3.Add new tab popup will be displayed and select the dynamics 365 app
4.Edit the environment and dynamics 365 app
5.User will search with environment name, but screen reader search result will not announce results count even not announcing "no results found".
6.Same issue in dynamics 365 app search.
![image](https://user-images.githubusercontent.com/105775102/169277488-d9255c05-dc82-4acb-8f63-cf4f9e43ac93.png)
